services/app-api/forms/2026/qms/measureTemplates.ts

```diff
@@ -1108,7 +1109,6 @@ export const measureTemplates: Record<
       {
         type: ElementType.NdrEnhanced,
         id: "measure-rates",
-        label: "Performance Rates",
```

Comment:
This was removed due to NdrEnhanced already defaults to `Performance Rate`, and passing in `Performance Rates` duplicates the trailing s.